### PR TITLE
Fixed all issues cited in the last PR review

### PR DESCRIPTION
**Pull request by felixfavour**
- Changed const to let
- and others...

### CODE REVIEW

File: dataservice.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        combinedProductMap.set(product.id, product);
+    }
+};
+// Function to return the result of above function
+exports.getCombinedProductMap = () => {
+    if (combinedProductMap.size === 0) {
+        combineProductsWithCategories()
+    }
+    return combinedProductMap;
+};
```
Comment (suthagar23) [inline dataservice.js]: Need a new line at last

File: dataservice.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+            if(product.categoryId  === category.id) {
+                product.categoryName = category.categoryName
+            }
+        }
+        combinedProductMap.set(product.id, product);
+    }
+};
+// Function to return the result of above function
+exports.getCombinedProductMap = () => {
+    if (combinedProductMap.size === 0) {
```
Comment (suthagar23) [inline dataservice.js]: Can't we normalize this line?

File: server.js [hunk 3]
```diff
@@ -1,49 +1,42 @@
...
-const app = express()
+const express = require('express');
+const app = express();
 
 app.get('/info', (req, res) => {
-    info = [
-        {"serverName": "localhost5000"},
-        {"sampleName": "chinemerem-react"},
+    const info = [
+        {"serverName": "chinemerem-react"},
```
Comment (suthagar23) [inline server.js]: Get the server name from pacakge.json

Comment (suthagar23) [reply server.js]: Server version also

File: dataservice.js (hunk 1 above)
Comment (felixfavour) [reply dataservice.js]: Sorry? I didn't catch that.

**Pull request closed without merging**